openedx/openedx-platform
# [WIP][BB-3676] Add support for ar_SA language

This PR works towards adding support for the Arabic(Saudi Arabia) language.

**JIRA tickets**: [BB-3676](https://tasks.opencraft.com/browse/BB-3676)

~**Screenshots**:~

**Sandbox URL**: TBD 

**Testing instructions**:

1. Pull the branch
2. `make lms-static`
3. `make lms-restart`
4. you can either change the language in `/update_lang` or change the `LANGUAGE_CODE` to `ar-sa`
5. The language should be changed

**Author notes and concerns**:

None

**Reviewers**
